scipy/stats/tests/test_stats.py

```diff
@@ -3243,6 +3244,19 @@ def test_result_attributes(self):
         attributes = ('statistic', 'pvalue')
         check_named_results(res, attributes)
 
+    @dec.knownfailureif(True, "Tough NIST Test Cases fail")
```

Comment:
Here's the fix: https://github.com/rgommers/scipy/commit/2d480cec2d49a3b176551c5f17fa0249f5749db4
Can you take that commit over in this PR @aeklant?

The one test case that still needs slightly reduced tolerance (``rtol=1e-4``) is really pathological - large array with only 5 unique values which have a relative difference between them of order 1e-8. I don't think it's worth spending time on improving numerical accuracy for that case further.